modules/apps/account/account-service/src/main/java/com/liferay/account/service/impl/AccountGroupRelLocalServiceImpl.java

```diff
@@ -39,37 +40,22 @@
 public class AccountGroupRelLocalServiceImpl
 	extends AccountGroupRelLocalServiceBaseImpl {
 
+	/**
+	 * @deprecated As of Cavanaugh (7.4.x), replaced by {@link #addAccountGroupRel(long, String, long)}
+	 */
+	@Deprecated
```

Comment:
I think we can just remove all the deprecated methods since `AccountGroupAccountEntryRel` was never released, and this can be considered as a new service.